test/functional/dataElements/xdmObject/xdmObjectEditing.spec.js

```diff
@@ -0,0 +1,454 @@
+
+  await extensionViewController.expectIsValid();
+  await extensionViewController.expectSettingsToContainData({
+    _unifiedjsqeonly: {
+      vendor: {
+        isLicensed: true
+      }
+    }
+  });
+});
+
+test("initializes form fields with boolean constant value of true", async () => {
+  await initializeExtensionView({
+    settings: {
+      schema,
+      data: {
+        _unifiedjsqeonly: {
+          vendor: {
+            isLicensed: true
+          }
+        }
+      }
+    }
+  });
+  await xdmTree.node("_unifiedjsqeonly").toggleExpansion();
+  await xdmTree.node("vendor").toggleExpansion();
+  await xdmTree.node("isLicensed").click();
+  await booleanEdit.expectConstantTrueValue();
+});
+
+test("allows user to select false constant value for property with boolean type", async () => {
+  const extensionViewController = await initializeExtensionView();
+  await xdmTree.node("_unifiedjsqeonly").toggleExpansion();
+  await xdmTree.node("vendor").toggleExpansion();
+  await xdmTree.node("isLicensed").click();
+  await booleanEdit.selectConstantInputMethod();
+  await booleanEdit.selectConstantFalseValueField();
+
+  await extensionViewController.expectIsValid();
+  await extensionViewController.expectSettingsToContainData({
+    _unifiedjsqeonly: {
+      vendor: {
+        isLicensed: false
+      }
+    }
+  });
+});
+
+test("initializes form fields with boolean constant value of false", async () => {
+  await initializeExtensionView({
+    settings: {
+      schema,
+      data: {
+        _unifiedjsqeonly: {
+          vendor: {
+            isLicensed: false
+          }
+        }
+      }
+    }
+  });
+  await xdmTree.node("_unifiedjsqeonly").toggleExpansion();
+  await xdmTree.node("vendor").toggleExpansion();
+  await xdmTree.node("isLicensed").click();
+  await booleanEdit.expectConstantFalseValue();
+});
+
+test("allows user to select no constant value for property with boolean type", async () => {
+  const extensionViewController = await initializeExtensionView();
+  await xdmTree.node("_unifiedjsqeonly").toggleExpansion();
+  await xdmTree.node("vendor").toggleExpansion();
+  await xdmTree.node("isLicensed").click();
+  await booleanEdit.selectConstantInputMethod();
+  await booleanEdit.selectConstantNoValueField();
+
+  await extensionViewController.expectIsValid();
+  await extensionViewController.expectSettingsToContainData({});
+});
+
+test("disables auto-populated fields", async () => {
+  await initializeExtensionView();
+  await xdmTree.node("_id").click();
+  await stringEdit.expectNotExists();
+});
+
+test("doesn't allow you to edit _id", async () => {
+  await initializeExtensionView();
+  await xdmTree.node("_id").click();
+  await stringEdit.expectNotExists();
+});
+
+test("allows you to edit context fields", async () => {
+  await initializeExtensionView();
+  await xdmTree.node("environment").toggleExpansion();
+  await xdmTree.node("type").click();
+  await stringEdit.expectExists();
+});
```

Comment:
This files was just a rename from xdmObject.spec.js. As far as I recall, there were no significant changes--mostly just changes to reflect the new xdmTree helper structure.